### PR TITLE
Add quickstart guide

### LINKED ISSUE #224: Add quickstart section to the documentation
Add content to https://emcee.readthedocs.io/en/stable/tutorials/quickstart/#quickstart section.

Please consult this [guide](https://emcee.readthedocs.io/en/stable/tutorials/quickstart/#quickstart) for a good example. At minimum show off basic loading, crossmatching, query and saving functionality.

### PR DESCRIPTION
**Pull request by camposandro**
Adds the quickstart guide which exemplifies the usage of:

- `from_dataframe`, to create an object catalog with a handful of points
- `read_hipscat`, to Ioad a catalog in HiPSCat format
- `crossmatch`, to perform a K nearest neighbors crossmatch between two catalogs
- `to_hipscat`, to save the resulting catalog to disk

Closes #224.

### CODE REVIEW

File: docs/gettingstarted/quickstart.ipynb [hunk 1]
```diff
@@ -2,24 +2,175 @@
  "cells": [
```
Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #3.    # The coordinates (ra, dec) for our objects of interest
</pre><p>Mention that they are in units of degrees</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:2]: <p>I would change <code>In this case</code> to something more like <code></code>Becuase we are downloading from the web source we need to install an additional package....<code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:2]: <p>Would it be possible to import the full catalog, but then do that trick that you just implemented, by having a small box?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:2]: <p>change "in the small" to "from the small"   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:2]: <p>explain a little bit more how it was lazy until now, and it got execute with compute</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:2]: <p>Just one more cell above showing query functionality.</p><p> </p><p>For instance, Andy had </p>

<pre class="ql-syntax" spellcheck="false">all_sky_object = gaia.crossmatch(ztf).query(
    "nobs_g_ztf_dr14 &gt; 50 and nobs_r_ztf_dr14 &gt; 50 and \
    parallax_gaia &gt; 0 and parallax_over_error_gaia &gt; 5 and \
    teff_gspphot_gaia &gt; 5380 and teff_gspphot_gaia &lt; 7220 and logg_gspphot_gaia &gt; 4.5 \
    and logg_gspphot_gaia &lt; 4.72 and classprob_dsc_combmod_star_gaia &gt; 0.5")
</pre><p>something like that, just much simpler.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (camposandro) [reply docs/gettingstarted/quickstart.ipynb:2]: Yes, I added a pointer to the full Gaia catalog and read from a small cone around the objects of interest!

File: docs/gettingstarted/quickstart.ipynb [hunk 2]
```diff
@@ -89,23 +89,35 @@
     "!pip install aiohttp --quiet"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "611cd2f6",
+   "metadata": {},
+   "source": [
+    "For the purpose of this tutorial we will only read a small 1 degree cone region of Gaia DR3, one that should contain our objects. While LSDB typically loads only the minimal amount of data it needs for our workflow, manually providing it with spatial information helps it identify which files to search for on disk."
```
Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:97]: loads -> loads into memory, right?. It is a bit unclear, we use verbs like load and read in a bit of a unclear fashion (all of us).

File: docs/gettingstarted/quickstart.ipynb [hunk 3]
```diff
@@ -115,17 +127,33 @@
...
-    ")"
+    "result = my_object_catalog.crossmatch(gaia, n_neighbors=1, radius_arcsec=1 * 3600, require_right_margin=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "44651e88",
+   "metadata": {},
+   "source": [
+    "Now let's say we wish to apply a cut to our data to get all the objects with a mean magnitude in the G band greater than 18 and a total number of observations AL greater than 200. We can build a query expression and filter the catalog. Because of the crossmatch, the name of the Gaia columns for the query need to contain the name of the catalog as a suffix, `_gaia`."
```
Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:138]: AL? I'm not sure I understand what happened here.

File: docs/gettingstarted/quickstart.ipynb [hunk 4]
```diff
@@ -115,17 +127,33 @@
...
+    "result = result.query(\"phot_g_mean_mag_gaia > 18 and astrometric_n_obs_al_gaia > 200\")"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "35994f0d",
    "metadata": {},
    "source": [
-    "We obtained at most 1 match for each of our initial objects."
+    "Our workflow takes advantage of Dask's \"lazy\" evaluation. This means that we have been defining a set of tasks which will only be executed at our command. When that happens, data will be loaded into disk and operations will be distributed among the several works for parallel computation. To trigger this we will call `compute` on the catalog that resulted from the crossmatch. The final result will be presented in a Pandas DataFrame."
```
Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:156]: Great, just "several works" -> "several workers"

**Review by nevencaplar — COMMENTED** [review]
Few more super small changes! I think it looks very good!

File: docs/gettingstarted/quickstart.ipynb (hunk 4 above)
Comment (camposandro) [reply docs/gettingstarted/quickstart.ipynb:156]: Thanks, nice catch!

File: docs/gettingstarted/quickstart.ipynb (hunk 2 above)
Comment (camposandro) [reply docs/gettingstarted/quickstart.ipynb:97]: You're right, I unified the terms and clarified this sentence!

File: docs/gettingstarted/quickstart.ipynb [hunk 5]
```diff
@@ -94,7 +94,7 @@
    "id": "611cd2f6",
    "metadata": {},
    "source": [
-    "For the purpose of this tutorial we will only read a small 1 degree cone region of Gaia DR3, one that should contain our objects. While LSDB typically loads only the minimal amount of data it needs for our workflow, manually providing it with spatial information helps it identify which files to search for on disk."
+    "On this tutorial we will read a small 1 degree cone region of Gaia DR3, one that should contain our objects. While LSDB typically reads into memory only the minimal amount of data it needs for our workflow, manually providing it with spatial information helps it identify which files to search for on disk."
```
Comment (nevencaplar) [inline docs/gettingstarted/quickstart.ipynb:97]: In this tutorial

**Review by nevencaplar — APPROVED** [review]

**Pull request merged**